newIDE/app/src/InstancesEditor/SelectionRectangle.js

```diff
@@ -4,6 +4,39 @@ import Rectangle from '../Utils/Rectangle';
 import { type InstanceMeasurer } from './InstancesRenderer';
 const gd: libGDevelop = global.gd;
 
+const getRectangleNormalizedBoundaries = ({
```

Comment:
Might be more efficient (see other comments) to pass a rectangle that you mutate directly.